Omit state transfer log on debug (#706)
planetarium/libplanet: CHANGES.md

    `Swarm<T>` became to call `preloadBlockDownloadFailed` event handler
    taken as an argument.  If the event handler is not present, `Swarm<T>`
    throws `SwarmException`.  [[#694]]

>  -  `Swarm<T>` became prints less logs during transfer its states on debug level.

Reviewer: ```suggestion
 -  `Swarm<T>` became to print less logs on debug level during sending states.
```

(In order to make it clear if sender prints these logs or receiver does.)
